storage/data-plane/src/test/java/com/microsoft/azure/storage/TransferManagerTest.groovy

```diff
@@ -749,8 +757,10 @@ class TransferManagerTest extends APISpec {
         10 * 1024 * 1024  | 1 * 1024 * 1024   | 5        || 10
         10 * 1024 * 1024  | 1 * 1024 * 1024   | 10       || 10
         500 * 1024 * 1024 | 100 * 1024 * 1024 | 2        || 5
-        500 * 1024 * 1024 | 100 * 1024 * 1024 | 4        || 5
         10 * 1024 * 1024  | 3 * 512 * 1024    | 3        || 7
+        // 500 * 1024 * 1024   | 100 * 1024 * 1024 | 4        || 5
+        // Disabling this test, as it causes Java Out of Memory error on Dev Ops VM.
+        // Will enable it later, when a decision on virtual machines upgrade is made.
```

Comment:
File an issue (if one isn't filed), and be sure to reference it from here.

Reply:
Added the link in the comment for reference.

Reply:
Commented on the issue, but we should be able to change the numbers here to not overload the vm. There's nothing inherently special about them that we can't get from some slightly smaller numbers.

Reply:
Yup as suggested, I updated the numbers and test ran fine in dev ops.